client/lib/plugins/wporg-data/test/mocks/local-store.js

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
Thanks for refactoring those mocks. They look way much cleaner now, and also tests are easier to read 👍